### DISCUSSION TITLE
Raspi OS Lite?

### DISCUSSION THREAD

**Discussion by DCElfving** [Geektoolkit/Dynaframe3#150] [Q&A]
Hey there – Dynaframe looks to be exactly what I need for a project. 

Will it run on Raspberry Pi OS Lite, or is it necessary to install the full desktop version? I'm trying to run it via ssh on a Raspberry Pi 3B, and getting segmentation faults.

Thanks!

**Comment by Geektoolkit** [comment]
I haven't tried it on lite, only full desktop.  Segmentation faults can occur if the images you're trying to load are too large.  I'd try with 2 1080p simple images in a folder inside of the Pictures folder, and see if you can get that to launch. If not you may have to use the full pi build to get it running. I've had it running on a pi3b so I know it's possible, but I've also had alot of segfaults when using too many photos, or too large of ones.

**Reply by DCElfving** [reply]
Ah. I'm trying to play back a series of 1024x1024 videos in a directory. I can get it working with a shell script and omxplayer, but not without pausing between each video.

**Comment by Geektoolkit** [comment]
I've never tried it at that resolution.  Dynaframe adds on a media stack, a webserver (though a light one), and a playlist manager on top of what omxplayer does. I've found usually the pi runs out of memory on a segfault, or linux forcefully shuts it down if the memory pressure gets too great. Also if you're running it via SSH the display: may be getting confused. Please try hooking a keyboard/mouse directly up and launching it via the console, that should at least narrow down if it's because of SSH or not. Also try just images and see if that works, if that works through SSH it may help to debug where the issue is.

and I'd try a basic 1080p video.  Then you'll have enough data to narrow things down fully. If you can watch the memory to see if it's getting to 1.5 gigs or so. The 3b only has 2 gigs I believe which isn't much...I was segfaulting all over the place with the A+ I was trying to use
